zebedee-cms/src/main/java/com/github/onsdigital/zebedee/util/slack/PostMessageAttachment.java

```diff
@@ -0,0 +1,42 @@
+package com.github.onsdigital.zebedee.util.slack;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PostMessageAttachment {
+    public enum Color {
+        Good,
```

Comment:
Same here - upper case and consider adding the string value to the enum and using that in your constructor below.